src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (sourceString) => {
+  const splittedSource = sourceString.split(';');
+  const resultObj = {};
```

Comment:
The variable name 'resultObj' is too generic and does not describe its purpose. A more descriptive name like 'styleObject' would improve readability.